presto-cli/src/main/java/io/prestosql/cli/AlignedTablePrinter.java

```diff
@@ -18,6 +18,7 @@
 import com.google.common.collect.ImmutableList;
```

Comment:
do you need to add product tests for cli for that?

Reply:
> do you need to add product tests for cli for that?


i do not feel this is necessary, why?